### ISSUE TITLE
Doesn't work with deadpool_redis

### ISSUE THREAD

**Issue by xerenahmed** [daniel7grant/redis-macros#1]
I'm using deadpool_redis for asynchronous connection pool. But implementations doesn't fit for this crate.

```rs
error[E0277]: the trait bound `Lecture: deadpool_redis::redis::FromRedisValue` is not satisfied
   --> src/graphql/lecture.rs:23:18
    |
23  |                 .query_async(&mut conn)
    |                  ^^^^^^^^^^^ the trait `deadpool_redis::redis::FromRedisValue` is not implemented for `Lecture`
```

**Comment by daniel7grant** [comment]
I'm not familiar with `deadpool_redis`, are its traits compatible with the `redis` crates traits? If they are, the code probably can be copied over from the `redis` implementation.

**Comment by xerenahmed** [comment]
Yes, it re-exports the redis crate with some additional functions of its own.

**Comment by daniel7grant** [comment]
I added a fix in version 0.2.0, so now the macro uses the `redis` crate in scope. See the [documentation](https://github.com/daniel7grant/redis-macros/blob/master/README.md#using-deadpool-redis-or-other-crates) and the [derive_deadpool example](https://github.com/daniel7grant/redis-macros/blob/master/examples/derive_deadpool.rs), but most likely this will fix your problem:

```rust
use deadpool_redis::redis;
```
